### ISSUE TITLE
Plugin stays green unless all of the instances fail

### ISSUE THREAD

**Issue by mkolar** [pyblish/pyblish-lite#60]
We have validators that work on multiple instances, however if one instance fails and other passess, it doesn't show red in the GUI, publishing stops correctly and terminal prints the error message, but artists are confused for the lack of visual feedback. 

On top of that, if this happens the plugins doesn't offer actions that should appear on failed state (for repairing the error for example).

**Comment by mottosso** [comment]
I just noticed a similar bug.

- #61

**Comment by tokejepsen** [comment]
@mkolar any update on this? Encountering the problem currently.

**Comment by mkolar** [comment]
Hey no no updates. I didn't have any development time on my hands, due to other projects. We're working around it for now, kinda by making sure, validators pass :) 

Good training for people to clean their scene, even though it's mostly just frustrating.

**Comment by tokejepsen** [comment]
Having a look at this now, and here is a reproducible for this and #61 ;

```python
import pyblish_lite
import pyblish.api


class Collect(pyblish.api.ContextPlugin):

    order = pyblish.api.CollectorOrder

    def process(self, context):

        context.create_instance(name="A")
        context.create_instance(name="B")


class ValidateFail(pyblish.api.InstancePlugin):

    order = pyblish.api.ValidatorOrder

    def process(self, instance):

        if str(instance) == "A":
            assert False, "Failing instance A"


class ValidateSuccess(pyblish.api.InstancePlugin):

    order = pyblish.api.ValidatorOrder + 0.1

    def process(self, instance):

        self.log.info("Succeeding.")


pyblish.api.register_plugin(Collect)
pyblish.api.register_plugin(ValidateFail)
pyblish.api.register_plugin(ValidateSuccess)

pyblish_lite.show()
```

**Comment by tokejepsen** [comment]
Fixed with #75
